common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -661,26 +700,35 @@ div.video {
     }
 
     ol.subtitles {
-        width: 0;
-        height: 0;
+      width: 0;
+      height: 0;
     }
 
     ol.subtitles.html5 {
-        @extend %ui-depth0;
-        background-color: rgba(243, 243, 243, 0.8);
-        height: 100%;
-        position: absolute;
-        right: 0;
-        bottom: 0;
-        top: 0;
-        width: 275px;
-        padding: 0 $baseline;
-        display: none;
+      @extend %ui-depth0;
+      background-color: rgba(243, 243, 243, 0.8);
+      height: 100%;
+      position: absolute;
+      right: 0;
+      bottom: 0;
+      top: 0;
+      width: 275px;
+      padding: 0 $baseline;
+      display: none;
+    }
+
+    a.hide-captions {
+      i:before {
+        content: "\f0d9"; // Overriding `.fa-caret-right` to make it `.fa-caret-left`
```

Comment:
I guess you could also do this:
    -moz-transform: scaleX(-1);
    -o-transform: scaleX(-1);
    -webkit-transform: scaleX(-1);
    transform: scaleX(-1);